neo4j/v1/session.py

```diff
@@ -120,7 +102,7 @@ def session(self):
         return session
 
     def recycle(self, session):
```

Comment:
Realized that session.close already does this :) Still unsure if this should be public and why it's on GraphDatabase, but again, unrelated to this PR.